docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,819 @@
+
+```css
+.electronics .electronics\:\!bg-primary-600 {
+    --tw-bg-opacity: 1 !important;
+    background-color: rgb(46 106 230 / 1 !important) !important;
+}
+```
+
+This ensures the styles only apply when two conditions are met: the parent element must have the `.electronics` class and the target element must have the `electronics:!bg-primary-600` class.
+::
+
+## Setting Up the Storefront
+
+Now you need to configure your Storefront to send the appropriate config ID to the Middleware based on the current URL path.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+In your Next.js SDK configuration, implement the `getConfigSwitcherHeader` function:
+
+```ts
+// apps/storefront-unified-nextjs/sdk/config.ts
+import { defineSdkConfig, getPathnameFromRequestHeaders } from '@vue-storefront/next';
+import type { UnifiedEndpoints } from 'storefront-middleware/types';
+
+function getConfigSwitcherHeader(baseHeaders: Record<string, string>) {
+  const pathname =
+    typeof window !== 'undefined' ? window.location.pathname : getPathnameFromRequestHeaders(baseHeaders);
+  
+  // Determine the config ID based on URL path
+  return pathname?.includes('/electronics') ? 'electronics' : 'apparel';
+}
+
+export function getSdkConfig() {
+  return defineSdkConfig(({ buildModule, config, getRequestHeaders, middlewareModule }) => ({
+    unified: buildModule(middlewareModule<UnifiedEndpoints>, {
+      apiUrl: `${config.apiUrl}/commerce/unified`,
+      defaultRequestConfig: {
+        getConfigSwitcherHeader,
+        headers: getRequestHeaders,
+      },
+      // ... other config
+    }),
+    // ... other modules
+  }));
+}
+```
+
+#tab-2
+In your Nuxt SDK configuration, define a reusable `getConfigSwitcherHeader` function:
+
+```ts
+// apps/storefront-unified-nuxt/sdk.config.ts
+import type { CommerceEndpoints, UnifiedCmsEndpoints, UnifiedEndpoints } from 'storefront-middleware/types';
+
+export default defineSdkConfig(({ buildModule, config, getRequestHeaders, middlewareModule }) => {
+  const route = useRoute();
+  
+  // Create a reusable function for determining the config ID
+  function getConfigSwitcherHeader() {
+    return route.path.includes('/electronics') ? 'electronics' : 'apparel';
+  }
+  
+  return {
+    // Unified commerce module
+    unified: buildModule(middlewareModule<UnifiedEndpoints>, {
+      apiUrl: `${config.apiUrl}/commerce/unified`,
+      defaultRequestConfig: {
+        getConfigSwitcherHeader,
+        headers: getRequestHeaders(),
+      },
+      // ... other config
+    }),
+    // ... other modules
+  };
+});
+```
+::
+
+## Updating the File Structure for Path Routing
+
+To enable path-based routing, you need to restructure your application to include a dynamic `[configId]` parameter in the route. This allows you to access the store identifier directly from the URL.
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+For Next.js apps using the App Router, update your file structure:
```

Comment:
nice to have: seems like these file structure adjustments could be done with a single `mv` command run and we could add it to the guide here